src/cascadia/LocalTests_TerminalApp/SettingsTests.cpp

```diff
@@ -7,6 +7,7 @@
 #include "../TerminalApp/CascadiaSettings.h"
 #include "JsonTestClass.h"
 #include <defaults.h>
+#include "../ut_app/TestDynamicProfileGenerator.h"

```

Comment:
Should we converge these tests at some point?

Reply:
There's #3838 already, which includes converging these two projects.